src/rct2/S6Exporter.cpp

```diff
@@ -562,6 +562,7 @@ extern "C"
         SDL_WriteU8(rw, gCheatsDisableLittering);
         SDL_WriteU8(rw, gCheatsNeverendingMarketing);
         SDL_WriteU8(rw, gCheatsFreezeClimate);
+	SDL_WriteU8(rw, gCheatsDisablePlantAging);
```

Comment:
I think when changing these, you need to increment the network protocol version. (Also, the indentation doesn't match the lines above, they're using spaces)